server/public/js/usersrequest.js

```diff
@@ -0,0 +1,179 @@
+/* eslint-disable */
+
+const getUserRequests = () => {
+  const createNode = element => document.createElement(element);
+  const append = (parent, el) => parent.appendChild(el);
+
+  const formatDate = (date) => {
+    const day = new Date(date).getDate();
+    const month = new Date(date).getMonth() + 1;
+    const year = new Date(date).getFullYear();
+
+    const createdAt = `${day}/${month}/${year}`;
+    return createdAt;
+  };
+
+  const retrievedToken = localStorage.getItem('token');
+
+  const url = 'https://maintenance-tracker-andela.herokuapp.com/api/v1/users/requests/';
+  // const url = 'http://localhost:8000/api/v1/users/requests/';
+
+  fetch(url, {
+    method: 'get',
+    mode: 'cors',
+    headers: {
+      'x-auth': retrievedToken,
+    },
+  })
+    .then(res => res.json())
+    .then((userRequests) => {
+      const requests = userRequests.data.request;
+
+      return requests.map((request) => {
```

Comment:
Since you are having a similar code somewhere, is it possible to abstract this into it's own function?

Reply:
Yes they look similar, but the issue with making it into a function is that I'm having different columns for each table